package.json

```diff
@@ -380,6 +380,11 @@
         "command": "powerpages.actionsHub.refresh",
         "title": "%powerpages.actionsHub.refresh.title%",
         "icon": "$(refresh)"
+      },
+      {
+        "command": "powerpages.actionsHub.switchEnvironment",
```

Comment:
Let's use the hierarchy like `microsoft.powerplatform.pages.actionsHub.switchEnvironment`